app/src/main/java/practicum/getfitla_v3/NutritionDetailActivity.java

```diff
@@ -1,34 +1,53 @@
 package practicum.getfitla_v3;
 
-import android.content.Intent;
 import android.os.Bundle;
 import android.support.v7.app.AppCompatActivity;
-import android.view.View;
+import android.widget.TextView;
+
 //Please ignore this page for now, this is a dummy page
-public class NutritionDetailActivity extends AppCompatActivity{
+public class NutritionDetailActivity extends AppCompatActivity {
+    private int passedid;
```

Comment:
Only declare what you need to show in the view. The only variables you need are passedname and passedprocess